react/features/mobile/react-native-sdk/middleware.js

```diff
@@ -56,5 +62,20 @@ const externalAPIEnabled = isExternalAPIAvailable();
     }
 
     return result;
-}
+});
+
+/**
+ * Check if native modules are being used or not.
+ */
+!externalAPIEnabled && StateListenerRegistry.register(
+    state => state['features/base/conference'].conference,
+    (conference, previousConference) => {
+        if (conference !== previousConference) {
```

Comment:
Re-write these so we abort if there is no new conference, and launch if there is a new one but there isn't a previous one.

If we jump from one conference to another we need to abort then launch.